src/meta/src/barrier/command.rs

```diff
@@ -64,8 +64,8 @@ pub enum Command {
 }
 
 impl Command {
-    pub fn checkpoint() -> Self {
-        Self::Plain(Mutation::Nothing(NothingMutation {}))
+    pub fn checkpoint(mutation: Option<Mutation>) -> Self {
+        Self::Plain(mutation.unwrap_or(Mutation::Nothing(NothingMutation {})))
```

Comment:
Remove `Option`, use `Mutation::Nothing` by default.

Reply:
Will fix with Mutation and Barrier refactor in next PR. 😂

Reply:
Fixed with `plain_with` instead, keep `checkpoint` with nothing mutation.